### PR TITLE
Resolved #540 & #541

### LINKED ISSUE #540: Remove speakers:  Sherman Chen & Subhankar Shah
they won't be able to make it. so please help me remove them! Thank you!

### PR DESCRIPTION
**Pull request by naokichau**
Preview: http://naokichau.github.io/2018.fossasia.org
Resolved #540 & #541

### CODE REVIEW

**Comment by naokichau** [comment]
#540

**Comment by naokichau** [comment]
also resolved #541

File: index.html [hunk 1]
```diff
@@ -4996,7 +5020,10 @@ <h1>Speakers</h1>
                         </div>
                         <div class="speaker-bio-tooltip-pin"></div>
                         <div class="speaker-bio-tooltip">
-                            <div class="speaker-bio-tooltip-content">Saptak Sengupta is an developer and enthusiast, who weirdly loves frontend development as much as he loves backend development. He works mainly in python, javascript, and PHP. Besides, he has been contributing to various open source organizations like FOSSASIA, jQuery, etc. He loves fiddling around with devops and infrastructure stuff.</div>
```
Comment (duttaditya18) [inline index.html]: Why was this edited?

File: index.html [hunk 2]
```diff
@@ -956,8 +1021,11 @@ <h1>Conference Program
                             <li>
                                 <div class="schedule-title">
                                     <span class="time">Saturday, March 24, 10:00 AM - 7:00 PM</span>
-                                     <span class="title">
-                                        <a href="./event/tracks.html#2018-03-24">Keynotes, Business and Technology Tracks, Young Developers Day, Group Photo, UNESCO Hackathon</a></span>
```
Comment (duttaditya18) [inline index.html]: Why was this editted?

**Comment by naokichau** [comment]
yep, I use a beautify tool in my code editor 😄

**Pull request merged**